<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Cantarell</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <strikeout>false</strikeout>
    <stylestrategy>PreferDefault</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>ImageSteganography</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ImageSteganography.qrc">
    <normaloff>:/UI/MainWindow/Graphics/WindowIcon</normaloff>:/UI/MainWindow/Graphics/WindowIcon</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>559</width>
     <height>340</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="centralLayout" stretch="0,0">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="encryptButton">
        <property name="statusTip">
         <string>Encrypt the message to the loaded image.</string>
        </property>
        <property name="text">
         <string>&amp;Encrypt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decryptButton">
        <property name="statusTip">
         <string>Decrypt the message from the loaded image.</string>
        </property>
        <property name="text">
         <string>&amp;Decrypt</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="statusTip">
         <string>Load an image from the disk.</string>
        </property>
        <property name="text">
         <string>&amp;Load Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="statusTip">
         <string>Save the image to the disk.</string>
        </property>
        <property name="text">
         <string>&amp;Save Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="bottomLayout">
      <item>
       <widget class="QLabel" name="messageLabel">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>&amp;Message</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="buddy">
         <cstring>messageEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="messageEdit">
        <property name="statusTip">
         <string/>
        </property>
        <property name="tabChangesFocus">
         <bool>false</bool>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="graphicLabel">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="graphicView">
        <property name="text">
         <string>No image is loaded from the disk.</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuAlgorithms">
     <property name="statusTip">
      <string>Select an image steganography algorithm.</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Algorithms</string>
     </property>
     <property name="icon">
      <iconset resource="ImageSteganography.qrc">
       <normaloff>:/UI/MainWindow/Graphics/AlgorithmsIcon</normaloff>:/UI/MainWindow/Graphics/AlgorithmsIcon</iconset>
     </property>
     <addaction name="actionLSB"/>
    </widget>
    <addaction name="menuAlgorithms"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionLSB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LSB</string>
   </property>
   <property name="statusTip">
    <string>Least Significant Bit Algorithm.</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="ImageSteganography.qrc">
     <normaloff>:/UI/MainWindow/Graphics/AboutIcon</normaloff>:/UI/MainWindow/Graphics/AboutIcon</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Show the about window related to this program.</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+A</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="ImageSteganography.qrc">
     <normaloff>:/UI/MainWindow/Graphics/HelpIcon</normaloff>:/UI/MainWindow/Graphics/HelpIcon</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="statusTip">
    <string>Open help documentation.</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>messageEdit</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>encryptButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>decryptButton</tabstop>
 </tabstops>
 <resources>
  <include location="ImageSteganography.qrc"/>
 </resources>
 <connections/>
</ui>
